<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aplicación de escritorio</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(250, 250, 250)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="bg_img">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/ABSTRACT_COLORFUL_BACKGROUND_01.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>261</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(245, 245, 245);
</string>
    </property>
    <property name="title">
     <string>Formulario de envío</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>261</width>
       <height>211</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Uso de CPU</string>
      </attribute>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Dank Mono</family>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(13, 71, 161);
color:rgb(250, 250, 250);
</string>
       </property>
       <property name="text">
        <string>Cantidad de procesos</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="get_cpu">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>142</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
border-color: blue;
</string>
       </property>
      </widget>
      <widget class="QPushButton" name="enviar_cpu">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>140</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(250, 250, 250);
background-color: rgb(26, 35, 126);
rounded-corner: 0px;
border-radius:0px;</string>
       </property>
       <property name="text">
        <string>Enviar</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Uso de Memoria</string>
      </attribute>
      <widget class="QLineEdit" name="get_memoria">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>142</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
border-color: blue;
</string>
       </property>
      </widget>
      <widget class="QPushButton" name="enviar_memoria">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>140</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(250, 250, 250);
background-color: rgb(26, 35, 126);
rounded-corner: 0px;
border-radius:0px;</string>
       </property>
       <property name="text">
        <string>Enviar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Dank Mono</family>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(74, 20, 140);
color:rgb(250, 250, 250);
</string>
       </property>
       <property name="text">
        <string>Cantidad de procesos</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
